controlplane/package.json

```diff
@@ -64,6 +65,7 @@
     "pg-boss": "^9.0.3",
     "pino": "^8.16.1",
     "postgres": "^3.4.3",
+    "postmark": "^3.11.0",
```

Comment:
This must be vendor neutral. Email should be sent over SMTP. You can use https://github.com/nodemailer/nodemailer/

Reply:
Done